04-Multi-Band-Raster.Rmd

```diff
@@ -294,30 +307,74 @@ plotRGB(RGB_stack,
 plotRGB(RGB_stack,
         r = 1, g = 2, b = 3, 
         scale=800,
-        stretch = "Lin")
+        stretch = "lin")
+```
 
+In the first image the forest was a nice bright green, however, seeing individual
+features was a bit of the challenge.  In the second plot with `stretch="lin"`, 
+the forest isn't as pretty green but individual features are easier to see. In 
+the upper left corner an area of uniform green can now be differentiated into 
+seperate trees or shrubs. 
 
-```
+>Bonus: Here we did a linear ("lin") stretch of the values. You can also stretch 
```

Comment:
hmmmm yea i'm not sure about this section. this image isn't idea to teach stretch as the stretch is pretty good without using the stretch function. some of me wonders if we just remove the discussion of stretch altogether???